core/modules/taxonomy/taxonomy.admin.inc

```diff
@@ -597,7 +597,7 @@ function taxonomy_overview_terms($form, &$form_state, TaxonomyVocabulary $vocabu
   $form['#forward_step'] = $forward_step;
   $form['#empty_text'] = t('No terms available.');
   if (taxonomy_vocabulary_access('create', $vocabulary)) {
-    $form['#empty_text'] = t('No terms available. <a href="@link">Add term</a>.', array('@link' => url('admin/structure/taxonomy/' . $vocabulary->machine_name . '/add')));
+    $form['#empty_text'] .= ' ' . l(t('Add term'), 'admin/structure/taxonomy/' . $vocabulary->machine_name . '/add');
```

Comment:
The string `No terms available.` is added to `$form['#empty_text'] `a couple of lines above. No need to be double-translating it. This change basically adds only `No terms available.` as a to-be-translated string, since `Add term` is already translated (from the existing "+ Add term" link).